docs/mp/grpc/server-side-services.adoc

```diff
@@ -212,8 +236,8 @@ public Stream<ResponseType> invoke(RequestType req)
 As with unary methods, the Helidon gRPC MP API supports multiple different method signatures for implementing server streaming
 methods.
 
-=== ClientStreaming Methods
-A client streaming method receives a stream of requests from the client and when the request stream is complete it sends back a
+==== ClientStreaming Methods
+A client streaming method receives a stream of requests from the client and when the request stream is complete, it sends back a
 response. A traditional gRPC Java client streaming method takes two `StreamObserver` parameters, one is the stream of client
 requests and the other is used to send back the single response in the same way that a unary method sends a response. As with
 unary methods Helidon gRPC MP supports different method signatures for client streaming methods.
```

Comment:
As with unary methods, Helidon gRPC MP supports...

Reply:
Already updated in in PR https://github.com/oracle/helidon/pull/4606